### DISCUSSION TITLE
Is there a way to remove "id" from selected fields?

### DISCUSSION THREAD

**Discussion by rwjack** [SigNoz/signoz#2386] [Q&A] [answered]
Clicking the "X" does nothing.

![image](https://user-images.githubusercontent.com/59068073/221409143-f3625253-d359-4661-bfa9-4d1ef01c5a55.png)

**Comment by pranay01** [comment]
@nityanandagohain Do you have more insights here?

**Comment by nityanandagohain** [comment]
Thanks for the question, as of now it is not possible but we will come up with a solution.

**Comment by nityanandagohain** [comment, marked as answer]
Progress can be tracked on this issue https://github.com/SigNoz/signoz/issues/2397

**Reply by rwjack** [reply]
Great, thanks!
